src/sentry/lang/javascript/processor.py

```diff
@@ -578,15 +614,16 @@ def expand_frames(self, frames, release):
                     })
 
                 # Store original data in annotation
+                # HACK(dcramer): we stuff things into raw which gets popped off
+                # later when adding the raw_stacktrace attribute.
+                raw_frame = frame.to_json()
                 frame.data = {
-                    'orig_lineno': frame.lineno,
```

Comment:
Couldn't we derive this data out of the raw stacktrace? I'm not sure if it's necessary in both places though, that's more of a product decision.